### PR TITLE
Show variations in sgf review

### PR DESCRIPTION
**Pull request by icehong**
I made some changes of  Review mode ,  add two button for navigation from different variations.  
But I don't know how to create vector icons.  Maybe you can add two more icons for this funcion.

### CODE REVIEW

File: android/src/main/java/org/ligi/gobandroid_hd/ui/fragments/NavigationFragment.java [hunk 1]
```diff
@@ -29,9 +29,15 @@
...
+    @Bind(R.id.btn_last_var)
+    ImageView last_var_btn;
+
+    @Bind(R.id.btn_next_var)
+    ImageView next_var_btn;
+
     @OnClick(R.id.btn_next)
     public void gameNavNext() {
-        GameForwardAlert.Companion.showIfNeeded(getActivity(), game);
+        game.redo(0);
```
Comment (ligi) [inline android/src/main/java/org/ligi/gobandroid_hd/ui/fragments/NavigationFragment.java]: how do people now access all variations?

**Comment by icehong** [comment]
'Next move' will goto the first variation on default. If there is more than one variations, at the same time 'next variation' will be shown. we can switch from different variations from 'last' and 'next' variations button.

**Comment by ligi** [comment]
Sorry but I do not think this solution is intuitive - also I do not know what Icons to use - so I have to respectfully decline this PR

**Comment by icehong** [comment]
I don't konw why you keep #108 as an enhencement and did not accecpt the patch.  This is implemnt the same way as gogui, qGo.    I don't think showing a popup window is a usual way to navigation from different variations.    Anyway , it's up you :) .

**Comment by ligi** [comment]
I agree that the popup is the ideal way - but with buttons like this it is not intuitive to understand what is going on and leads to bad UX IMHO

**Comment by icehong** [comment]
@ligi  ,  could you tell me where to get those icon's svg code ?   I want to see is there are same icons are suitable for  'next branch' and 'last branch' ,  or just the same as gogui icon : 
gogui-downgogui-up

**Comment by ligi** [comment]
Sorry I cannot - I got the other icons from: https://design.google.com/icons/

**Comment by icehong** [comment]
I made some change for this :
1. add two more icons . 
2. making buttons gray when the were disabled.   
Hopes you will changes your mind. 
device-2016-03-19-011008

**Comment by ligi** [comment]
As an option ( switchable in settings - default off ) I would accept this.  But directly confronting the user with this - without explaining it through the settings I still don't like the idea

**Comment by ildar** [comment]
It's no problem understanding if a user had used other go programs before.
I vote for this patch: it's painful to review sgfs in the way it is now.
Sorry to intrude...

**Comment by ligi** [comment]
But we cannot take for granted that users used other go programs before - your vote is noted but this is not a democracy - I still stand that I only merge this if it is behind a setting

**Comment by icehong** [comment]
I can understand that you want an good user experience, how about add a promote when an user first open sgf files in review mode,  we can give a tips to user to this new method,   and also keep and ' do not promote again' option to avoid annoying promote?

**Comment by ligi** [comment]
no - uses just click these dialogs away without reading

**Comment by icehong** [comment]
Add an option int settings and  variation window,  and default off.   As you wish.   

https://github.com/icehong/gobandroid/tree/variations

**Comment by ildar** [comment]
THANX!

**Pull request closed without merging**